tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/anndata.py

```diff
@@ -42,8 +42,9 @@ def open_anndata(
         if h5ad.schema_version == "":
             h5ad.schema_version = get_cellxgene_schema_version(ad)
         if h5ad.schema_version not in CXG_SCHEMA_VERSION_IMPORT:
-            logging.warning(f"H5AD has old schema version, skipping {h5ad.dataset_h5ad_path}")
-            continue
+            msg = f"H5AD has unsupported schema version {h5ad.dataset_h5ad_path}, expected {CXG_SCHEMA_VERSION}"
```

Comment:
```suggestion
            msg = f"H5AD has unsupported schema version {h5ad.schema_version}, expected {CXG_SCHEMA_VERSION}"
```

Reply:
I think we want all three (dataset_id, schema_version, expected schema version).  Will push a change.